feature/aks-cluster-creation/README.md

```diff
@@ -0,0 +1,55 @@
+## Terraform scripts to provision AKS cluster #
+***
+The Azure Kubernetes Service (AKS) is the service for deploying, managing, and scaling containerized applications with Kubernetes. This repo contain terraform files, to automate the AKS cluster deployment.
+***
+#### Nice to know:
+- AKS require two resources group. First resource group which is managed by user and contains only the Kubernetes service resource. Second resource group, known as the node resource group, contains all of the infrastructure resources associated with the cluster. These resources include the Kubernetes node VMs, virtual networking, and storage. AKS automatically remove the node resource group whenever the cluster is removed.

```

Comment:
I do not remember exactly how, but I was able to force AKS via terraform to do it all in one RG. Let's move it to separate task.

Reply:
Hymm.. intresting, I will take a look but I was sure that it is not possible (based on azure documentation): 

https://docs.microsoft.com/en-us/azure/aks/faq:
```
As you work with the node resource group, keep in mind that you **cannot**:

Specify an **existing** resource group for the node resource group.
```

Reply:
Checked once again, and in fact I was wrong. AKS creates separate RG for it's resources.